pkg/trace/config/config_test.go

```diff
@@ -33,7 +33,11 @@ func TestConfigHostname(t *testing.T) {
 		defer func() {
 			fallbackHostnameFunc = os.Hostname
 		}()
-		_, err := Load("./testdata/site_override.yaml")
+		config, _ := Load("./testdata/site_override.yaml")
+		// As the default path for `config.DDAgentBin` may exist, set `DDAgentBin` to an invalid path and call again config.validate().
```

Comment:
This explanation makes no sense.  Can you please clarify?

* Why are we using an invalid path? What's the problem if the path exists?
* What are we setting the hostname to empty?

Reply:
> This explanation makes no sense.

The issue is that `Load` is very hard / impossible to mock and should be rewritten in order to be tested (or this test should be removed)